### DISCUSSION TITLE
Route name in edge template

### DISCUSSION THREAD

**Discussion by iarkaroy** [adonisjs/core#2798] [Help] [answered]
How can I get the current route name in edge template? I am trying to conditionally display a section based on current route.

Core version: `5.1.8`

**Comment by thetutlage** [comment, marked as answer]
Not available directly. But you can grab it from ctx as follows.

```
request.ctx.route
```

This is what you get https://docs.adonisjs.com/guides/context#route

Another option is to share as a direct property via middleware

**Reply by iarkaroy** [reply]
This will do. Thanks.
